schema.graphql

```diff
@@ -1,3 +1,139 @@
+# ADD TOTAL TOKENS
+# spread fees are expresed in sell token
+# write a how to query on a specific block
+
+
+######################################################################################################################
+type CowSwap @entity {
+  "Singleton #1"
+  id: ID!
+  "Total token ever traded"
+  tokensTotal: BigInt!
+  "Total orders placed"
+  ordersTotal: BigInt!
+  "Total unique addresses on cowsap"
+  usersTotal: BigInt!
+  "Total different settlements"
+  settlementTotal: BigInt!
+  "Total traded amount in USD"
+  totalTradedAmountUsd: BigDecimal
+  "Total traded amount in ETH"
+  totalTradedAmountEth: BigDecimal
+  "Total fee amount in USD"
+  totalFeeAmountUsd: BigDecimal
+  "Total fee amount in Eth"
+  totalFeeAmountEth: BigDecimal
+}
+
+######################################################################################################################
+type CowSwapDayData @entity {
+  "Day timestamp"
+  id: ID!
+  "Start day timestamp"
+  timestamp: BigInt!
+  "Total token traded that day"
```

Comment:
OK, i'm not adding suggestions for the rest of items. I think u get it from the `Totals` entity